samples/openmp/Makefile

```diff
@@ -0,0 +1,28 @@
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+ifdef STRACE
+# OPTS = --strace
+endif
+
+all: appdir rootfs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+
+rootfs: appdir
+	sudo $(MYST) mkext2 appdir ext2rootfs
+	sudo chown $(USER).$(USER) ext2rootfs
+	$(MYST) fssig --roothash ext2rootfs > roothash
+	# $(MYST) mkcpio appdir rootfs
```

Comment:
Why don't we use CPIO archive?

Reply:
I prefer using EXT2 rootfs to reduce EPC consumption.

Reply:
For this size of applications, EXT2fs and ramfs probably don't have much performance difference.

Reply:
@jxyang , any specific concern about using EXT2fs instead of CPIO for this sample?

Reply:
Just curious because ext2fs requires more step in the makefile and uses one more `myst` argument.  But I can live with that if you prefer.